pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -266,6 +266,7 @@ let
     , license
     , readmeFile ? "README_${dictFileName}.txt"
     , sourceRoot ? dictFileName
+    , maintainers ? with maintainers; [ vlaci ]
```

Comment:
Sorry again...

```suggestion
    , maintainers ? with lib.maintainers; [ vlaci ]
```

Reply:
No need to be sorry, let's get this fixed!